src/directives/formio.js

```diff
@@ -38,6 +38,69 @@ module.exports = function() {
           $scope._src += 'live=1';
         }
 
+        // Build the display map.
+        $scope.show = {};
+        var boolean = {
+          'true': true,
+          'false': false
+        };
+
+        var submission = $scope._submission || {data: {}};
+        var updateComponents = function() {
+          // Change the visibility for the component with the given key
+          var updateVisiblity = function(key) {
+            $element
```

Comment:
That is what I tried initially, but I could not get the ng-show attribute to interpret the value of show.key or show[key]. It was as if $scope.show here, was not in the scope, of the individual fields (Additionally a $scope.$apply() wouldn't connect the two).

Any clues?